api/README.md

```diff
@@ -4,4 +4,8 @@ This is a **low-level, unstable** API for programmatically accessing the Nexus V
 
 The roadmap is for it to be supplanted by a more strategically designed and consistent Nexus SDK. The API will likely shift as part of the SDK development effort.
 
-The API is relatively self-documenting, see `./src/lib.rs`. Examples of using the API are given in `./examples`.
+The API is relatively self-documenting, see `./src/lib.rs`. Examples of using the API are given in `./examples`. The examples can be run with, for instance:
```

Comment:
This should document this step: https://github.com/nexus-xyz/nexus-zkvm/blob/main/api/examples/prover_run.rs#L17 as well.

Reply:
odd, I didn't have to do anything for that. I'll experiment.

Reply:
You probably already did it implicitly to run the examples through the CLI, but that's not always true.

Reply:
right, so my experiment is to try this on a fresh checkout.

Reply:
Indeed a step is missing
```
error loading and parsing RISC-V instruction: IOError(Os { code: 2, kind: NotFound, message: "No such file or directory" })
```

Reply:
`cargo build -r` at the project root directory doesn't fix this.

`cargo build -r` in `<project_root>/examples` fixes this.

Reply:
So I added the necessary step. Tried that on another fresh checkout and it worked.